l1-contracts/contracts/common/OwnableRandomStorage.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.7.0) (access/Ownable.sol)
+
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Provides information about the current execution context, including the
+ * sender of the transaction and its data. While these are generally available
+ * via msg.sender and msg.data, they should not be accessed in such a direct
+ * manner, since when dealing with meta-transactions the account sending and
+ * paying for execution may not be the actual sender (as far as an application
+ * is concerned).
+ *
+ * This contract is only required for intermediate, library-like contracts.
+ */
+abstract contract Context {
+    function _msgSender() internal view virtual returns (address) {
+        return msg.sender;
+    }
+
+    function _msgData() internal view virtual returns (bytes calldata) {
+        return msg.data;
+    }
+}
+
+/**
+ * @dev Contract module which provides a basic access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * By default, the owner account will be the one that deploys the contract. This
+ * can later be changed with {transferOwnership}.
+ *
+ * This module is used through inheritance. It will make available the modifier
+ * `onlyOwner`, which can be applied to your functions to restrict their use to
+ * the owner.
+ */
+abstract contract OwnableRandomStorage is Context {
+    /// @dev Address of owner variable.
+    /// @dev Flag is placed at random memory location to not interfere with Storage contract.
+    // keccak256("owner") - 1;
+    uint256 private constant OWNER_ADDRESS = 0x02016836a56b71f0d02689e69e326f4f4c1b9057164ef592671cf0d37c8040bf;
```

Comment:
Just because we want to save auditors time :)
```suggestion
    uint256 private constant OWNER_ADDRESS = uint256(keccak256("owner")) - 1;
```

Reply:
I got this error: 
![Screenshot 2024-01-12 at 13 46 44](https://github.com/matter-labs/era-contracts/assets/34402761/44326110-d417-4fd8-98f2-0cfcc364bd1b)